src/storage/redis_metadata.h

```diff
@@ -143,6 +144,7 @@ class Metadata {
   virtual void Encode(std::string *dst);
   virtual rocksdb::Status Decode(const std::string &bytes);
   bool operator==(const Metadata &that) const;
+  virtual ~Metadata() = default;
```

Comment:
Oh previously this might work, because seems we didn't `delete Metadata` when subclass like `ListMetadata` is used. However I think we'd better has this.